wordle-web/pages/wordStats.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <!--create a leader board that shows the stats for the last ten words-->
+  <v-card>
+    <v-card-title class="text-center">Last Ten Word Stats</v-card-title>
+    <v-table>
+      <thead>
+        <tr>
+          <th>Date</th>
+          <th>Average Guesses</th>
+          <th>Average Time</th>
+          <th>Total Times Played</th>
+          <th>Total Wins</th>
+          <th>Total Losses</th>
+          <th>Did you play?</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="stats in statsToDisplay" :key="stats.date">
+          <td v-if="stats.date">{{ stats.date }}</td> <td v-else>No Data</td>
+          <td v-if="stats.averageGuesses">{{ stats.averageGuesses }}</td> <td v-else>No Data</td>
+          <td v-if="stats.averageTime">{{ stats.averageTime }}</td> <td v-else>No Data</td>
+          <td v-if="stats.totalTimesPlayed">{{ stats.totalTimesPlayed }}</td> <td v-else>No Data</td>
+          <td v-if="stats.totalWins">{{ stats.totalWins }}</td> <td v-else>No Data</td>
+          <td v-if="stats.totalWins && stats.totalTimesPlayed">{{ stats.totalTimesPlayed - stats.totalWins }}</td> <td v-else>No Data</td>
+          <td v-if="stats.played == true"> Yes </td> <td v-else-if="stats.played == false"> No </td> <td v-else>No Data</td>
```

Comment:
```suggestion
          <td> {{ stats.played ? "Yes" : "No" }} </td>
```
We don't need to conditionally render the elements if they are going to be the same,, we can just make the content conditional.